dashboard/db/migrate/20161007172920_rename_pd_district_payment_term_index.rb

```diff
@@ -0,0 +1,7 @@
+class RenamePdDistrictPaymentTermIndex < ActiveRecord::Migration[5.0]
+  def change
+    rename_index :pd_district_payment_terms,
+      'index_pd_district_payment_terms_on_school_district_id_and_course',
+      'index_pd_district_payment_terms-school_district_course'
```

Comment:
It seems odd that we are using a hyphen rather than an underscore, given that no other index (presumably, I didn't confirm) uses a hyphen. Consider changing to an underscore.